runtime/doc/api.txt

```diff
@@ -2995,15 +3024,17 @@ nvim_open_win({buffer}, {enter}, {*config})                  *nvim_open_win()*
                                   the number of chars are less than eight,
                                   they will be repeated. Thus an ASCII border
                                   could be specified as [ "/", "-", "\\", "|"
-                                  ], or all chars the same as [ "x" ]. An
-                                  empty string can be used to turn off a
-                                  specific border, for instance, [ "", "", "",
-                                  ">", "", "", "", "<" ] will only make
-                                  vertical borders but not horizontal ones. By
-                                  default, `FloatBorder` highlight is used,
-                                  which links to `VertSplit` when not defined.
-                                  It could also be specified by character: [
-                                  {"+", "MyCorner"}, {"x", "MyBorder"} ].
+                                  ],     or all chars the same as       [ "x"
+                                  ].     An empty string can be used to turn
+                                  off a specific border, for instance,       [
+                                  "", "", "", ">", "", "", "", "<" ]     will
+                                  only make vertical borders but not
+                                  horizontal ones.     By default,
+                                  <tt>FloatBorder</tt> highlight is used,
```

Comment:
Ok after a while I figured it out, in my system and yours doxygen or gen_vimdoc is producing different output than CI. 

Specifically on this `<tt>` marks and also a lot of clobbered white space as you can see in this diff. And ci is failing because gen_vimdoc is actually working fine there and removing those issues.

Reply:
lmao how did you even figure this out? Anyway, good job :)